src/useToken.js

```diff
@@ -0,0 +1,11 @@
+const getToken = () => {
+  const tokenString = localStorage.getItem('token')
+  return JSON.parse(tokenString)
+}
+const saveToken = userToken => {
+  localStorage.setItem('token', JSON.stringify(userToken))
+}
+export {
+  getToken,
+  saveToken
+}
```

Comment:
should add a blank line between functions and the export section.

Reply:
![image](https://user-images.githubusercontent.com/62316127/125916747-7a747571-cb8a-4f22-9530-da17d60c1046.png)